Dm 374 as a c8y user i should ensure that the documentation created by the earlier user stories are integrated and pushed to the develop (#45)
SoftwareAG/c8y-releasenotes: content/release-10-15-0/whatsnew-10-15-0.md

In an effort to provide you a more consistent experience across the platform the **All devices** page has been re-shaped. It is now implemented in the data grid fashion you already are familiar with from **Subassets** and **Search** pages. This offers improved appearence in various viewport sizes, easier filtering and sorting, pagination option (only for users with global inventory access) and sets solid ground for further improvements.

![All devices](/images/release-notes/devmgmt-devices-alldevices.png)

### Multiple LNS connection for Sigfox and Actility

The Sigfox and Actility agents now support the configuration of multiple connections per tenant. One connection can be associated with multiple devices. Each connection can be updated and deleted.
<br>

> For details, refer to

Reviewer: ```suggestion
For details, refer to [Sigfox](https://cumulocity.com/guides/protocol-integration/sigfox) or [Actility LoRa](https://cumulocity.com/guides/protocol-integration/lora-actility) in the *Protocol integration guide*.
```

> <br> Sigfox : https://cumulocity.com/guides/protocol-integration/sigfox

Reviewer: ```suggestion
```

> <br> Actility : https://cumulocity.com/guides/protocol-integration/lora-actility

Reviewer: ```suggestion
```

In an effort to provide you a more consistent experience across the platform the **All devices** page has been re-shaped. It is now implemented in the data grid fashion you already are familiar with from **Subassets** and **Search** pages. This offers improved appearence in various viewport sizes, easier filtering and sorting, pagination option (only for users with global inventory access) and sets solid ground for further improvements.

![All devices](/images/release-notes/devmgmt-devices-alldevices.png)

> ### Multiple LNS Connection for Sigfox and Actility

Reviewer: ```suggestion
### Multiple LNS connection for Sigfox and Actility
```

> The Sigfox and Actility agent now supports configuration of multiple connections per tenant. One connection can be associated with multiple devices. Each connection can be updated and deleted.

Reviewer: ```suggestion
The Sigfox and Actility agents now support the configuration of multiple connections per tenant. One connection can be associated with multiple devices. Each connection can be updated and deleted.
```

> 

Reviewer: Please add a link to the documentation: "For details, refer to []()".

Author: Done.
